README.md

```diff
@@ -11,43 +11,64 @@ adhocracy+ is designed to make online participation easy and accessible to every
 
 ## Installation for development
 
-### Requirements:
+### Requirements
 
  * nodejs (+ npm)
  * python 3.x (+ venv + pip)
  * libpq (only if postgres should be used)
+ * redis (only if celery is used)
 
-### Installation:
+### Installation
 
     git clone https://github.com/liqd/adhocracy-plus.git
     cd adhocracy-plus
     make install
     make fixtures
 
-### Start virtual environment:
+### Start virtual environment
+
     source venv/bin/activate
 
-### Check if tests work:
+### Check if tests work
 
     make test
 
-### Start a local server:
+### Start a local server
+
     make watch
 
-### Use postgresql database for testing:
+### Use postgresql database for testing
+
 run the following command once:
 ```
-make create-postgres
+make postgres-create
 ```
-to start the testserver with postgresql, run:
+to start the test server with postgresql, run:
 ```
 export DATABASE=postgresql
-make start-postgres
+make postgres-start
 make watch
 ```
 
 Go to http://localhost:8004/ and login with admin@liqd.net | password
 
+### Use Celery for task queues
+
+To start a celery worker node in the foreground, call:
```

Comment:
Add here what you have in docs/celery.md about celery eager should be false in local settings. Otherwise the worker won't do anything

Reply:
good point, have added missing instructions.